pkg/config/analysis/analyzers/util/service_lookup.go

```diff
@@ -61,6 +62,17 @@ func InitServiceEntryHostMap(ctx analysis.Context) map[ScopedFqdn]*v1alpha3.Serv
 		return true
 	})
 
+	// use meshConfig.trustDomain changed the default domain
+	// todo: replace by `values.global.proxy.clusterDomain`
+	ctx.ForEach(gvk.MeshConfig, func(r *resource.Instance) bool {
```

Comment:
can we get MeshConfig with this function? I am not sure, MeshConig is from configmap

Reply:
there's a hack for MeshConfig.

Reply:
Yes, the configmap was saved in cache as MeshConfig for analysis during initialization.